src/kafka-util/src/client.rs

```diff
@@ -817,7 +817,7 @@ impl TimeoutConfig {
         };
 
         let progress_record_fetch_timeout = if progress_record_fetch_timeout < transaction_timeout {
-            error!(
+            warn!(
```

Comment:
@petrosagg what do you think about this? Should we instead adjust the progress record fetch timeout in LaunchDarkly to be 10m for all the organizations where we've overridden the transaction timeout?

Reply:
I think the problem here is that the code defaults to 60s unconditionally even when no override exists for the progress fetch timeout. IMO the correct behavior would be that the default progress record fetch timeout is always derived from whatever value the transaction timeout is set to and if there exists an override in LD which is below the transaction limit then we error, because now we truly have misconfigured the system.

I put up a PR to this effect here https://github.com/MaterializeInc/materialize/pull/29322